common/gl_util/obj.hpp

```diff
@@ -0,0 +1,254 @@
+#pragma once
```

Comment:
Sorry, forgot to add this in the first review.

If I pass an invalid path for the obj data in `opt` mode, nothing gets rendered (rightfully so), but there aren't any warnings or errors to indicate why. All the error checking in this file that results in a **catastrophic** failure should result in a print. That is, those prints should always happen and not be guarded by NDEBUG. This is the scheme that we use elsewhere in ILLIXR as well.

Reply:
Done.